public/index.html

```diff
@@ -98,7 +98,7 @@ <h2>PayJoin here to open these channels</h2>
 					return;
 				}
 				channels.removeChild(channels.children[channels.children.length - 1]);
```

Comment:
Here I was thinking channels was immutable, forgetting we're in js land
```suggestion
				channels.removeChild(channels.lastChild);
```